optimum/habana/trl/trainer/ppo_trainer.py

```diff
@@ -820,10 +913,24 @@ def train_minibatch(
             old_logprobs, values, logits, vpreds, logprobs, mask, advantages, returns
         )
         loss = loss_p + loss_v
-        self.accelerator.backward(loss)
+        global _recorded_graph
+
+        if _recorded_graph is None:
+            _recorded_graph = ht.hpu.HPUGraph()
+            s = ht.hpu.default_stream()
+
+            with ht.hpu.stream(s):
+                _recorded_graph.capture_begin()
+                self.accelerator.backward(loss)
+                _recorded_graph.capture_end()
+        else:
+            ht.core.mark_step()
```

Comment:
Replay will have internal mark_step, this mark_step is not required.

Reply:
removed